scripts/build-dependencies.sh

```diff
@@ -1,3 +1,8 @@
+if [ ! -d ./libphonenumber/.git ]; then
```

Comment:
this means if the libphonenumber folder is not cached/corrupted, download from git,

we should not do this, we should always download libphonenumber